devices/RgbDiode/RgbDiode.cs

```diff
@@ -194,10 +189,10 @@ private static void GuardFactor(params double[] factors)
             }
         }
 
-        private static void SetValue(PwmChannel channel, double value, bool inverse)
+        private static void SetValue(PwmChannel channel, double value, LedType ledType)
         {
             var dc = value / 255;
```

Comment:
detail: place this in the if as you're not using other wise.

Reply:
![image](https://github.com/nanoframework/nanoFramework.IoT.Device/assets/19893965/ec0346ca-65fe-4b94-b998-0bb64d51ae1d)
It is :)

Reply:
I need a new pair of eyes ;-)